system-addon/common/Reducers.jsm

```diff
@@ -29,7 +29,11 @@ const INITIAL_STATE = {
     // Have we received real data from history yet?
     initialized: false,
     // The history (and possibly default) links
-    rows: []
+    rows: [],
+    editForm: {
```

Comment:
Can you maybe add a comment this should be used in content only and not be synchronized?